bitnami/haproxy/templates/service.yaml

```
@@ -25,7 +25,8 @@ spec:
   externalTrafficPolicy: {{ .Values.service.externalTrafficPolicy | quote }}
   {{- end }}
   {{- if eq .Values.service.type "LoadBalancer" }}
-  loadBalancerSourceRanges: {{ .Values.service.loadBalancerSourceRanges }}
+  loadBalancerSourceRanges:   
+    {{ toYaml .Values.service.loadBalancerSourceRanges | nindent 4 }}
```

Could you please keep it in the same line? The `nindent` already adds the new line when rendering

Additionally, please add a condition for empty values:

```suggestion
  {{- if not (empty .Values.service.loadBalancerSourceRanges) }}
  loadBalancerSourceRanges: {{- toYaml .Values.service.loadBalancerSourceRanges | nindent 4 }}
  {{- end }}
```